test/lib/simpleRequire.js

```diff
@@ -142,7 +142,7 @@
                 var factoryRet = factory.apply(null, deps.map(function (dep) {
                     return dep === exportsPlaceholder ? modExports : dep;
                 }));
-                return factoryRet || modExports;
+                return factoryRet || modExports.exports || modExports;
```

Comment:
Could you explain this change to me? I think this is not same with the AMD spec

Reply:
Well, `modExports` would be an argument for `factory` function, as matter of fact, variable `modExports` will be variable `module` when function runs. Then as I have mention above, The `exports` attribute of `module` will have whatever  `echarts`  have exported. 
```js
modExports = {
    exports: echarts.
}
```